### PR TITLE
Prepare 2.8.7 release

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
https://devpi.net/ronny/dev/pytest will contain devpi tests

**Comment by nicoddemus** [comment]
:+1: Thanks for tackling this so quickly @RonnyPfannschmidt !

Btw, how did you test the wheel package using devpi? :grin:

File: doc/en/announce/release-2.8.7.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+pytest-2.8.6
+============
+
+pytest is a mature Python testing tool with more than a 1100 tests
+against itself, passing on many different interpreters and platforms.
```
Comment (nicoddemus) [inline doc/en/announce/release-2.8.7.rst]: I think it is worth mentioning it that such a short release was due to a serious regression

**Comment by The-Compiler** [comment]
LGTM!

**Comment by RonnyPfannschmidt** [comment]
@nicoddemus my devpi did it automatically,

however i did upload both formats in one go

**Comment by nicoddemus** [comment]
Hmmm so did I. Is it some configuration setting perhaps? Weird, should have worked... :grimacing:

**Comment by RonnyPfannschmidt** [comment]
Ftr, the new devpi on the windows box also didn't run bo the, @hpk42 ?

**Pull request merged**